plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/AddToCartWithOptions.php

```diff
@@ -167,25 +167,27 @@ protected function render( $attributes, $content, $block ) {
 			$default_quantity = apply_filters( 'woocommerce_add_to_cart_quantity', 1, $product->get_id() );
 
 			$context = array(
-				'productId' => $product->get_id(),
-				'quantity'  => $default_quantity,
+				'productId'   => $product->get_id(),
+				'productType' => $product->get_type(),
+				'quantity'    => $default_quantity,
 			);
 
-			if ( $product instanceof \WC_Product && $product->is_type( 'variable' ) ) {
+			if ( $product->is_type( 'variable' ) ) {
```

Comment:
We already checked that `$product` is an instance of `WC_Product` here:

https://github.com/woocommerce/woocommerce/blob/331801fadcb5de5cb6c94bc6ae53c5ca33fb5d69/plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/AddToCartWithOptions.php#L103

So there was no need to repeat the check.